packages/breadcrumbs/src/Breadcrumbs.ts

```diff
@@ -0,0 +1,294 @@
+/*
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import {
+    CSSResultArray,
+    html,
+    PropertyValues,
+    SpectrumElement,
+    TemplateResult,
+} from '@spectrum-web-components/base';
+import {
+    property,
+    query,
+    queryAssignedElements,
+    state,
+} from '@spectrum-web-components/base/src/decorators.js';
+import '@spectrum-web-components/icons-workflow/icons/sp-icon-folder-open.js';
+import '@spectrum-web-components/action-menu/sp-action-menu.js';
+import '@spectrum-web-components/menu/sp-menu-item.js';
+import { ActionMenu } from '@spectrum-web-components/action-menu';
+import {
+    BreadcrumbItem as BreadcrumbElement,
+    BreadcrumbSelectDetail,
+} from './BreadcrumbItem.js';
+import { createRef, Ref, ref } from 'lit/directives/ref.js';
+
+import styles from './breadcrumbs.css.js';
+import '../sp-breadcrumb-item.js';
+import { ifDefined } from '@spectrum-web-components/base/src/directives.js';
+
+const MAX_VISIBLE_ITEMS = 4;
+
+type BreadcrumbItem = {
+    label?: string;
+    href?: string;
+    value: string;
+    offsetWidth: number;
+    isVisible: boolean; // false if displayed in menu overlay
+};
+
+/**
+ * @element sp-breadcrumbs
+ *
+ * @slot icon - change the default icon of the action menu
+ * @slot root - Breadcrumb item to always display
+ * @slot - Breadcrumb items
+ */
+export class Breadcrumbs extends SpectrumElement {
+    public static override get styles(): CSSResultArray {
+        return [styles];
+    }
+
+    /**
+     * Always display the first breadcrumb
+     */
+    @property({ type: Boolean, attribute: 'show-root' })
+    public showRoot = false;
```

Comment:
Should we use this attribute, or provide the `slot="root"` directly for consumers?